src/emqx_mod_topic_metrics.erl

```diff
@@ -63,6 +63,9 @@
         , terminate/2
         ]).
 
+%% stats.
+-export([ last_stats/2 ]).
```

Comment:
`stats` is too big of a scope,
maybe we should expose it like the `rate` API, name it `rates` instead.